corehq/apps/domain/deletion.py

```diff
@@ -9,8 +11,11 @@
 from corehq.apps.accounting.models import Subscription
 from corehq.apps.accounting.utils import get_change_status
 from corehq.form_processor.interfaces.dbaccessors import CaseAccessors, FormAccessors
+from corehq.util.log import with_progress_bar
 from dimagi.utils.chunked import chunked
 
+logger = logging.getLogger(__name__)
```

Comment:
Where does this log end up?

Reply:
It gets outputted to the console when the ```delete_domain``` management command is run.  It should end up in one of the log files as well, but I'm not sure which.

Reply:
Haha fair enough. I remember looking for a log like this once and could not find it at all